Dockerfile

```diff
@@ -1,14 +1,28 @@
-FROM pangeo/base-image:2020.06.03
+FROM pangeo/base-image:2021.07.17
+
+# lifted from https://github.com/2i2c-org/pilot-hubs/blob/824d1dd53a6b5170a32a0458bcf2724f8a412a95/images/user/Dockerfile
+RUN apt-get update -qq --yes && \
+    apt-get install --yes --no-install-recommends -qq \
+        psmisc \
+        sudo \
+        libapparmor1 \
+        lsb-release \
+        libclang-dev  > /dev/null
```

Comment:
This may be failing due to being run as jovyan rather than root. These could also be moved to `apt.txt` which is run as root during `ONBUILD`.

Reply:
Do we want to combine this with the apt-get install above?

Reply:
I'm trying to avoid ONBUILD. It is working locally though. (To be honest I'm not checking the logs here yet. Still struggling to slim down the image and get the R-dev stuff in there.)

Reply:
Now that we're not based on Pangeo's image, we aren't dealing with the weirdness of ONBUILD.